### DISCUSSION TITLE
Lesson 13 React - Material-ui error -  'Box' cannot be used as a JSX component.

### DISCUSSION THREAD

**Discussion by Callassie** [smartcontractkit/full-blockchain-solidity-course-py#1393] [Q&A] [answered]
I am getting this error when adding the box to YourWallet. 

![screenShot1](https://user-images.githubusercontent.com/102427356/162877937-96dab2be-a930-4e3d-b816-0f97ccb46179.png)

Searching it seems to show that it is relatively new with an update from Material-ui. Some suggest adding "resolutions" to the package.json, which for me did not work.

Here is another link that I found discussing other components affected by this:

https://github.com/react-native-svg/react-native-svg/issues/1741

Here are my dependencies:

  "dependencies": {
    "@material-ui/core": "^4.12.4",
    "@testing-library/jest-dom": "^5.16.4",
    "@testing-library/react": "^12.1.4",
    "@testing-library/user-event": "^13.5.0",
    "@types/jest": "^27.4.1",
    "@types/node": "^16.11.26",
    "@types/react": "^17.0.44",
    "@types/react-dom": "^17.0.15",
    "@usedapp/core": "^0.12.8",
    "ethers": "^5.6.2",
    "react": "18.0.0",
    "react-dom": "^18.0.0",
    "react-scripts": "5.0.0",
    "source-map-resolve": "^0.6.0",
    "typescript": "^4.6.3",
    "web-vitals": "^2.1.4"
  },

Any suggestions?

**Comment by cromewar** [comment, marked as answer]
Hello @Callassie 
I also faced this changes when I was working on this project, I don't remember what I did tough... but if you want you can give a look to the project here:

https://github.com/cromewar/Full_Stack_DeFi_App
